### PR TITLE
Feature/registermatch

### PR DESCRIPTION
**Pull request by deeksha-developer**
tournament page and register match page , 
merge code tournament and registermatches , conflict solved and all merge code push to registermatch branch and solve 
slider change , pull the development branch add and conflict resolved
https://app.clickup.com/t/85ztp2b2p

### CODE REVIEW

File: src/app/globals.css
Comment (rajeshdeveloper99) [inline src/app/globals.css:1]: remove line 108 & 109

File: src/Components/Slider/Slider.tsx
Comment (rajeshdeveloper99) [inline src/Components/Slider/Slider.tsx:1]: remove this page

File: src/app/userDashboard/registerMatches/page.tsx [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+  mapType: string;
+  version: string;
+  date: string;
+  time: string;
+  lastServival: string;
+  roomId: string;
+  password: string;
+}
+
+const regMatches = () => {
```
Comment (rajeshdeveloper99) [inline src/app/userDashboard/registerMatches/page.tsx:21]: remove dummy data **"regMatches"**

**Review by rajeshdeveloper99 — COMMENTED** [review]
please check comments @deeksha-developer

**Review by expertdeveloper5821 — APPROVED** [review]

**Review by developerAkshay1 — APPROVED** [review]

**Pull request merged**